app/analysis/template.hbs

```diff
@@ -77,6 +77,7 @@
   <p>
     {{#each exportLinks as |link index|}}
       <button {{action 'download' link.url}}
+        type="button"
```

Comment:
Out of curiosity: Why specify the type in this case?

Reply:
Because I don't want any side effects like e.g pressing enter in the filter form and triggering a download with it.. IMHO submit buttons should only be allowed in forms and only one per form.

Reply:
Damn, i didn't know the default type is `submit`. Makes total sense.

Reply:
Yep that took me by surprise too.. I had a form which deleted entries when pressing enter.